.env.example

```diff
@@ -15,4 +15,5 @@ TIER_3_RATE_LIMIT =
 LIMIT_DURATION =
 HBAR_RATE_LIMIT_TINYBAR =
 HBAR_RATE_LIMIT_DURATION =
+RATE_LIMIT_DISABLED =
```

Comment:
Update the Helm chart also
See https://github.com/hashgraph/hedera-json-rpc-relay/pull/578/files as a reference